t/ui/01-list.t

```diff
@@ -256,12 +256,12 @@ subtest 'available comments shown' => sub {
             $driver->find_element('#job_99963 .fa-comment')->get_attribute('title'),
             '2 comments available',
             'available comments shown for running jobs'
-        );
+        );    # uncoverable statement
```

Comment:
Hm, why are we unconditionally skipping tests here anyway?

Reply:
Good question. It looks like it's never run and there's no way to enable it?

Reply:
The comment says "to imporove performance". I can try enabling those tests.

Reply:
I changed the checks to be actually executed and to verify the current behavior.